Tasks/Common/webdeployment-common-v2/xdttransformationutility.ts

```diff
@@ -124,7 +124,7 @@ export function specialXdtTransformation(rootFolder, transformConfig, sourceConf
     }
     
     if(!isTransformationApplied) {
-        tl.warning(tl.loc('FailedToApplyTransformation'));
+        tl.warning(tl.loc('FailedToApplySpecialTransformation'));
```

Comment:
for new task if no transformation happens, it will show warning followed by an error.